python/cudf/cudf/utils/utils.py

```diff
@@ -179,6 +179,27 @@ def compare_and_get_name(a, b):
     return None
 
 
+def get_null_series(size, dtype=np.bool):
+    """
+    Creates a null series of provided dtype and size
+
+    Parameters
+    ----------
+    size:  length of series
+    dtype: dtype of series to create; defaults to bool.
+
+    Returns
+    -------
+    a null cudf series of provided `size` and `dtype`
+    """
+    from cudf.utils import cudautils
+    from cudf.core import Series
+
+    device_array = cudautils.zeros(size, dtype=dtype)
+    mask = cudautils.zeros(len(device_array), dtype=np.uint8)
```

Comment:
Instead of both of these calls you can use:
```
cudf.core.column.column_empty(len(device_array), dtype, True)
```

and construct a Series from that column.

Reply:
Thanks for this , will incorporate this and update the function.